Use `strk` as a default fee token (#2749)
foundry-rs/starknet-foundry: CHANGELOG.md

## [Unreleased]

### Cast

#### Changed

> - `--fee-token` and `--version` flags are now optional, `strk` and `v3` will be inferred automatically

Reviewer: ```suggestion
- `--fee-token` and `--version` flags are now optional, `strk` and `v3` will be used by default
```

- Requirements validation during `snforge` runtime
- `snforge check-requirements` command

### Cast

#### Changed

- `--fee-token` and `--version` flags are now optional, `strk` and `v3` will be used by default

> ### `snforge_scarb_plugin`

Reviewer 2: nit:
```suggestion
```

#### Changed

- `snforge_scarb_plugin` will now also emit warnings when errors occur

### Cast

> #### Changed

Author: ```suggestion
#### Deprecated
```
